chart/templates/controller.yaml

```diff
@@ -187,6 +208,31 @@ spec:
         {{ end -}}
         securityContext:
           runAsUser: {{.ControllerUID}}
+      - name: sp-validator
```

Comment:
Per in-person discussion -- let's move this container to a separate deployment with it's own service account. I suggest creating a new chart file for this, something like `chart/templates/sp-validator.yaml`.